heroku_connect/management/commands/import_mappings.py

```diff
@@ -20,27 +20,43 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         connection_id = options.get('CONNECTION_ID', None)
-        app_name = options.get('HEROKU_APP', settings.HEROKU_CONNECT_APP_NAME)
-        if not (connection_id or app_name):
+        self.app_name = options.get('HEROKU_APP', settings.HEROKU_CONNECT_APP_NAME)
+        if not (connection_id or self.app_name):
             raise CommandError("You need ether specify the application name or "
                                "the connection ID.")
         if connection_id is None:
-            try:
-                connections = utils.get_connections(app_name)
-            except requests.HTTPError as e:
-                raise CommandError("Failed to load connections") from e
+            connections = self.get_connections()
             if len(connections) == 0:
-                raise CommandError("There are no connections associated "
-                                   "with the app '%s'." % app_name)
+                msg = self.style.NOTICE(
+                    "There is no connection associated with current user the app '%s'."
+                    " Trying to link the current user with Heroku Connect..." % self.app_name
```

Comment:
Do we owe the user a success message after "Trying ..."? To be able to remain silent on success, I think it's better to be affirmative and drop the ellipsis: 

* "Linking the current user with Heroku Connect.", or 
* "Ensuring that the current user is linked with Heroku Connect."

Reply:
True, I agree those messages should state a clean intent.